.github/workflows/ci.yml

```diff
@@ -54,23 +54,6 @@ jobs:
       - run: pip install -r requirements-dev.txt
       - run: mypy --config-file mypy.ini
 
-  build-docs:
```

Comment:
not sure I like this removal - imagine code changing a docstring in a way that doc build starts to fail, but _not_ touching the `docs/**` - then we would no longer find this out; IMHO we should have kept both sections, even at the cost of building the docs twice... or think about more precise conditions to trigger the docs build

Reply:
Good catch!

Made a fix for this in a separate PR: [#5998 ](https://github.com/modin-project/modin/pull/5998)